packages/core/src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -775,6 +775,35 @@ describe('StripeV3PaymentStrategy', () => {
                     expect(stripeV3JsMock.confirmCardPayment).toHaveBeenCalled();
                 });
 
+                it('throws stripe error when 3d auth fails', async () => {
+                    const threeDSecureRequiredErrorResponse = new RequestError(getResponse({
+                        ...getErrorPaymentResponseBody(),
+                        errors: [
+                            { code: 'three_d_secure_required' },
+                        ],
+                        three_ds_result: {
+                            token: 'token',
+                        },
+                    }));
+                    const authenticationFailErrorResponse = new RequestError(getResponse({
+                        ...getErrorPaymentResponseBody(),
+                        errors: [
+                            { code: 'payment_intent_authentication_failure' },
+                        ],
+                    }));
+                    const stripeErrorMessage = 'Stripe error message.';
+
+                    jest.spyOn(paymentActionCreator, 'submitPayment')
+                        .mockReturnValueOnce(of(createErrorAction(PaymentActionType.SubmitPaymentFailed, threeDSecureRequiredErrorResponse)))
+                        .mockReturnValueOnce(of(createErrorAction(PaymentActionType.SubmitPaymentFailed, authenticationFailErrorResponse)));
```

Comment:
I think this is not necessary. 🤔

Reply:
Thank you for the comments, already corrected them and commit them to this PR. Let me know if everything is correct 😎